tests/e2e/util.go

```diff
@@ -5647,3 +5646,20 @@ func startCSIPods(ctx context.Context, client clientset.Interface, csiReplicas i
 	isServiceStopped := false
 	return isServiceStopped, err
 }
+
+// waitForPodsToBeInRunningState function waits till all the pods comes up
+func waitForPodsToBeInRunningState(ctx context.Context, client clientset.Interface, namespace string,
+	statefulset *appsv1.StatefulSet) error {
+	waitErr := wait.Poll(healthStatusPollInterval, healthStatusPollTimeout, func() (bool, error) {
```

Comment:
dont use 
healthStatusPollInterval, healthStatusPollTimeout
if we have to change this variable for changes in volume health feature, it will impact this
use something like `poll * 2` if you need larger time vars

Reply:
removed.